Cargo.toml

```diff
@@ -435,7 +435,7 @@ zlog_settings = { path = "crates/zlog_settings" }
 
 agent-client-protocol = { version = "0.2.0-alpha.6", features = ["unstable"]}
 aho-corasick = "1.1"
-alacritty_terminal = { git = "https://github.com/zed-industries/alacritty.git", branch = "add-hush-login-flag" }
+alacritty_terminal = "0.25.1-rc1"
```

Comment:
I have bumped `alacritty_terminal` to 0.25.1-rc1 in #38505 since it fixes an issue alacritty had with not properly routing control characters to spawned processes.